lib/screen/onboarding_page.dart

```diff
@@ -62,7 +61,7 @@ class _OnboardingPageState extends State<OnboardingPage>
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
-    log("DefineViewRoutingEvent");
+    logger.info("DefineViewRoutingEvent");
```

Comment:
I think just update the package import to `import 'package:autonomy_flutter/util/log.dart'` and using `log.info` is fine